R/defineModel.R

```diff
@@ -0,0 +1,65 @@
+#' Defining the Model
+#'
+#' Give the complete formula and generate the expected kernel library.
+#'
+#' It processes data based on formula and label_names and creates a kernel
+#' library according to the parameters given in Kern_par.
+#'
+#' @param formula A symbolic description of the model to be fitted.
```

Comment:
For all argument description, make sure to mention data class name in the beginning of the description.

Example:
```
@param formula (formula) A symbolic description of the model to be fitted.
@param label_names (character) ....
@param data (dataframe) ....
@param kern_par (dataframe) ....
```